numpy/lib/function_base.py

```diff
@@ -4274,9 +4277,19 @@ def quantile(a,
     the same as the maximum if ``q=1.0``.
 
     The optional `method` parameter specifies the method to use when the
-    desired quantile lies between two data points ``i < j``.
-    If ``g`` is the fractional part of the index surrounded by ``i`` and ``j``,
-    and alpha and beta are correction constants modifying i and j:
+    desired quantile lies between two indexes ``i``  and ``j = i + 1 ``. 
+    In that case, we first  determine ``i + g``, a virtual index that lies 
+    between ``i`` and ``j`, where  ``i`` is the floor and ``g`` is the 
+    fractional part of the index. The final result is, then, an interpolation
+    of `a[i]` and `a[j]` based on ``g``. During  the computation of ``g``, ``i`` 
+    and ``j`` are modified using correction constants ``alpha`` and ``beta`` 
+    whose choices depend on the ``method`` used. Finally, note that since
+    python uses 0-based indexing, the code subtracts another 1 from the index 
+    internally.
+    
+    
+   The following formula determines the virtual index ``i + g``, the location
+   of the quantile in the sorted sample:
```

Comment:
These seem to have the wrong indent, I suspect this is the problem for why CI fails (also one blank line above only).

There is info on how to build the docs here: https://numpy.org/devdocs/dev/howto_build_docs.html (it can be a bit annoying to get running though).

You should be able to see `ci/circleci: build` at the bottom of the PR, if you click on details, it tells you what failed (this is the documentation build).
More importantly, if you click on the details of `ci/circleci: build artifact` you will get to the fully rendered version of the documentation with your changes!

In this case, you would see that the wrong indentation here, means that the whole block above is completely missing (which is why it reports an error I think).